pal/inc/pal.h

```diff
@@ -3830,6 +3830,8 @@ VirtualProtect(
            IN DWORD flNewProtect,
            OUT PDWORD lpflOldProtect);
 
+#define WRITE_WATCH_FLAG_RESET 1
```

Comment:
Could you please also move this to ```CommonPal.h``` (to reduce diff between official pal.h and ours)?